src/backend/joanie/tests/core/test_api_order.py

```diff
@@ -1331,13 +1343,26 @@ def test_api_order_create_payment(self, mock_create_payment, _mock_thumbnail):
                         "order_relations__position"
                     )
                 ],
+            },
+        )
+        with self.assertNumQueries(10):
+            response = self.client.patch(
+                f"/api/v1.0/orders/{order.id}/submit/",
+                data=data,
+                content_type="application/json",
+                HTTP_AUTHORIZATION=f"Bearer {token}",
+            )
```

Comment:
The docstring of this test, let's believe if an order is created by providing a billing address, the order state is automatically transitioned to `submitted` then the response return payment information.

But that's not the case, an extra request to submit the order must be sent.

Reply:
At first the docstring was right, but the behaviour changed and I forgot to change the docstring